### ISSUE TITLE
Undo of multiple row remove should call hook 'afterCreateRow' for each row removed

### ISSUE THREAD

**Issue by HaroldStruwig** [handsontable/handsontable#10219]
### Description
Undo of multiple row remove calls hook 'afterCreateRow' only once for the first row removed instead of multiple times for each row

### Steps to reproduce
1. Select multiple rows
2. Right click
3. Remove rows
4. Right click
5. Undo
6. Expect multiple calls to hook 'afterCreateRow' for each row index that was removed and now recreated again

### Demo
https://jsfiddle.net/f2bkatq1/2/

### Your environment
* Handsontable version: 12.3.0
* Browser name and version: Google Chrome 109
* Operating system: Windows 10

**Comment by adrianszymanski89** [comment]
Hi @HaroldStruwig 

Thank you for the suggestion. We have already reported an issue that touches on the same topic. The problem was already discussed, and that would require building a new feature, but it's not currently on our roadmap. I will close this issue and link it here: https://github.com/handsontable/handsontable/issues/6731

**Comment by HaroldStruwig** [comment]
@adrianszymanski89 I don't believe it is the same issue - I agree with what is stated in the reference you provided - that change will be very nice - but what I reported is something different which I believe is a bug - without this one can't rely on using 'afterCreateRow' only for knowing when new rows are created - we use it to assign each new entry to a DB so now we have missing entries.

**Comment by adrianszymanski89** [comment]
@HaroldStruwig 

You are right. This is more different than the mentioned issue than I initially thought. I will leave it as a separate issue, then.

**Comment by AMBudnik** [comment]
Hi @HaroldStruwig 

Could you tell us a bit more about

> without this one can't rely on using 'afterCreateRow' only for knowing when new rows are created - we use it to assign each new entry to a DB, so now we have missing entries.

The `afterCreateRow` hook specifies the starting index and the number of added rows. We can see the same behavior for the `alter()` method where we add new rows https://jsfiddle.net/doz92vaf/. In the demo, we add 5 rows and the hook is called only once but the 2nd parameter of the function returns `5`.

**Comment by HaroldStruwig** [comment]
Okay I forgot about the `number` parameter - probably because it was always 1 in the case of pasting a rows but the hook was called multiple times. CLOSING as this solves the issue.
